templates/taxi/driver_list.html

```diff
@@ -8,6 +9,19 @@ <h1>
       </a>
     </h1>
 
+    <form
+        action=""
+        method="get"
+        class="form-inline"
+    >
+      {{ search_form|crispy }}
+      <input
+          type="submit"
+          value="🔍"
+          class="btn btn-secondary"
+      >
+    </form>
```

Comment:
Move to a separate template and include where needed

Reply:
Done